src/main/java/frc/robot/RobotContainer.java

```diff
@@ -133,6 +135,7 @@ public class RobotContainer {
 
   /* Autos */
   private TaxiAuto taxiAuto = new TaxiAuto();
+  private AimAndFireRoutine aimAndFire = new AimAndFireRoutine();
```

Comment:
This isn't an auto, it should be up with commands. We're actually going to use this command a BUNCH in teleop.